railties/lib/rails/commands/server/server_command.rb

```diff
@@ -132,27 +155,34 @@ class ServerCommand < Base # :nodoc:
       def initialize(args = [], local_options = {}, config = {})
         @original_options = local_options
         super
-        @server = self.args.shift
```

Comment:
I think it would be better and clearer to do this than the `@_using` construct:

```ruby
super

deprecate_positional_using if @using
@using ||= options[:using]

@log_stdout = …
```